plugins/hydra_ray_launcher/example/conf/ray_mode/aws.yaml

```diff
@@ -0,0 +1,34 @@
+# @package _group_
+
+hydra:
+  launcher:
+    params:
+      sync_up:
+        # source dir is relative in this case, assuming you are running from
+        # <project_root>/hydra/plugins/hydra_ray_launcher/example
+        # absolute path is also supported.
+        source_dir: "."
+        # we leave target_dir to be null
+        # as a result the files will be synced to a temp dir on remote cluster.
+        # the temp dir will be cleaned up after the jobs are done.
+        # recommend to leave target_dir to be null if you are syncing code/artifacts to remote cluster so you don't need
+        # configure $PYTHONPATH on remote cluster
+        include: ["model", "*.py"]
+        # No need to sync up config files.
+        exclude: ["*"]
+      sync_down:
+        # relative path to the output of os.getcwd() on remote cluster
+        # for now we only support syncing from one dir.
+        # multirun is the default output dir for sweeper
+        source_dir: multirun
```

Comment:
multirun here looks like a flaw in the plugin , not a documentation issue.

Reply:
sorry for missing this one!

I don't think this is a design flaw with the plugin. It's more like a bad example app.

the source dir is where to download from remote machine. Since in our example app writes stuff in hydra's multirun dir on the remote machine, i think it makes sense to put multirun dir here.
I'm going to change the example app so we don't download multirun here, that way it's less confusing.

Reply:
All apps are writing to the configured multirun dir, aren't they?

Reply:
hmm ok, let's chat in person, probably more efficient :))

Reply:
per offline discussion, uses shouldnt need to set this in the common use case., we will get the dir from hydra config.